anthos-baremetal-edge-deployment/roles/abm-software/files/external-secrets-crd.yaml

```diff
@@ -12,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
----
+# [START anthosbaremetal_files_external_secrets_crd_customresourcedefinition_externalsecrets_kubernetes_client_io]
```

Comment:
we might want to very quickly check that there's no character length limit on region tags.

Reply:
104 characters 👍